client/extensions/woocommerce/components/address-view/index.js

```diff
@@ -117,7 +104,7 @@ class AddressView extends Component {
 					<FormSelect
 						name="country"
 						onChange={ onChange }
-						value={ country }
+						value={ country || 'US' }
```

Comment:
This is a good preventative idea. Let's do this for the state too. But we'll have to have a little upfront logic to support a good default for state - i.e. if country is falsy, and we're setting country to US then AL (Alabama) otherwise if country is not falsy, we should set AB (Alberta) for Canada or AL for US.

In my not-yet-landed PR we have a concept of default state, and if you make this change, I can update the logic with the default